poetry.toml

```diff
@@ -0,0 +1,2 @@
+[virtualenvs]
+prefer-active-python = true
```

Comment:
what does this option do exactly?

Reply:
Poetry can be installed in several ways and each one associates poetry with a specific version of python. When we run `poetry install`, poetry creates a virtual environment using the version of python it knows. 

This line makes poetry ignore its own version of python and use the version that is active at the moment to create the virtual environment.

This makes managing multiple versions of python easier.